rio_tiler/constants.py

```diff
@@ -14,6 +14,15 @@
 MAX_THREADS = int(os.environ.get("MAX_THREADS", multiprocessing.cpu_count() * 5))
 
 WEB_MERCATOR_CRS = CRS.from_epsg(3857)
+WEB_MERCATOR_OPP_CRS = CRS.from_string(
```

Comment:
I'm not sure to understand why this is needed?

Reply:
It is Opposite side CRS.
For details, please see follows.
https://github.com/cogeotiff/rio-tiler/pull/322#discussion_r547181980